tests/DecoderTests.elm

```diff
@@ -0,0 +1,81 @@
+module DecoderTests exposing(..)
+
+import Data.Decoder     exposing(decodeAudience)
+import Expect
+import Json.Decode      exposing(decodeString)
+import Test             exposing(..)
+
+
+
+audienceDecoderTests : Test
+audienceDecoderTests =
+    describe "Audience API decoder"
+        [ describe "valid audience types"
+              [ test "Authored" <|
+                    \_ ->
+                        Expect.ok <| decodeString decodeAudience """ {
```

Comment:
Nice addition to add Unit tests, but you could have gone a bit further and test against actually parsed valid values 😉 👍🏻 (instead of just `Expect.ok`)